src/test/resources/release-notes-file.md

```diff
@@ -0,0 +1,11 @@
+### New features
+
+* [JIRA-TICKET](jira-link): Some feature
```

Comment:
Not sure how you do the template replacement, but I'd try with some real examples, as they can mess with your regex (if you do any).